armi/reactor/blocks.py

```diff
@@ -1554,7 +1554,7 @@ def setAxialExpTargetComp(self, targetComponent):
 
             Sets the ``axialExpTargetComponent`` parameter on the block to the name of the Component
             which is passed in. This is then used by the
-            :py:class:`~armi.reactor.converters.axialExpansionChanger.AxialExpansionChanger`
+            :py:class:`~armi.reactor.converters.axialExpansionChanger.axialExpansionChanger.AxialExpansionChanger`
```

Comment:
Since these classes are imported into `armi.reactor.converters.axialExpansionChanger`, does this doc work?
```suggestion
            :py:class:`~armi.reactor.converters.axialExpansionChanger.AxialExpansionChanger`
```
The triplet `axialExpansionChanger.axialExpansionChanger.AxialExpansionChanger` is throwing me off

Reply:
> The triplet axialExpansionChanger.axialExpansionChanger.AxialExpansionChanger is throwing me off

🤢